Makefile

```
@@ -17,3 +17,13 @@ test:
 	$(MAKE) -C backend test
 	@echo "Running frontend tests..."
 	$(MAKE) -C frontend test
+
+diagrams:
```

checkmake (0.3.2)

[warning] Target body for "diagrams" exceeds allowed length of 5 lines (8).

(maxbodylength)